Fantasy-Quest/Assets/Scripts/DBubble/ShowBubble.cs

```diff
@@ -0,0 +1,106 @@
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace DBubble
+{
+    [AddComponentMenu("Scripts/DBubble/DBubble.ShowBubble")]
+    internal class ShowBubble : MonoBehaviour
+    {
+        [Header("Show Settings")]
+        [SerializeField]
+        private bool canShow = false;
+
+        [SerializeField]
+        private float lerpRate = 0.01f;
+
+        [Header("Sprites/Renderers")]
+        [SerializeField]
+        public List<Sprite> IconList = new List<Sprite>();
+
+        [SerializeField]
+        private SpriteRenderer bubbleSprite;
+
+        [SerializeField]
+        private SpriteRenderer iconSprite;
+
+        private void Awake()
+        {
+            bubbleSprite = GetComponent<SpriteRenderer>();
+            iconSprite = gameObject.transform.GetChild(0).GetComponent<SpriteRenderer>();
+            iconSprite.sprite = IconList[0];
```

Comment:
Missing `RequireComponent`, and you trying to get first child, it's not intuitive and uses magical number, move it to variables.